services/web/server/src/simcore_service_webserver/security/_authz_access_roles.py

```diff
@@ -92,6 +92,7 @@ class PermissionDict(TypedDict, total=False):
         can=[
             "product.details.*",
             "product.invitations.create",
+            "user.email.update",  # NOTE: keep until additional email feature (or equivalent) is added
```

Comment:
not sure that makes any sense but to you and @matusdrobuliak66

Reply:
Decided to remove the entrypoint instead of restricting the access. It is safer and the intention is more clear